src/api/controllers/credentials.ts

```diff
@@ -61,8 +61,59 @@ export class Credentials {
 		})
 	}
 
-	async issue_credentials(request: Request, user: GenericAuthUser, provider: string, subjectId?: string): Promise<Response> {
-		provider = provider.toLowerCase()
+	async issue_person_credential(user: GenericAuthUser, provider: string, subjectId?: string) {
+		const credential_subject: CredentialSubject = {
+			id: subjectId,
+			type: undefined
+		}
+
+		const credential = {
+			'@context': VC_CONTEXT.concat(VC_PERSON_CONTEXT),
+			type: ['Person', VC_TYPE],
+			issuanceDate: new Date().toISOString(),
+			credentialSubject: credential_subject,
+			'WebPage': [
```

Comment:
@DaevMithran did we decide to go with this format of `WebPage`? AFAIK, platforms like discord don't have a notion of `name` and `URL`. This is why we had the switch/case here before, to make the `WebPage` property adjust to the provider

Reply:
Sure got it, I'll change that, or is it better to remove the URL itself instead of hardcoding these?